runelite-client/src/main/java/net/runelite/client/plugins/microbot/util/magic/Rs2Magic.java

```diff
@@ -143,6 +156,10 @@ public static void superHeat(Rs2Item item, int sleepMin, int sleepMax) {
             sleep(50, 150);
             return Rs2Tab.getCurrentTab() == InterfaceTab.MAGIC;
         });
+        if (Rs2Widget.isWidgetVisible(218, 4) && Arrays.stream(Rs2Widget.getWidget(218, 4).getActions()).anyMatch(x -> x.equalsIgnoreCase("back"))){
```

Comment:
Is the MTA a custom interface for casting spells?

Reply:
And similar to the code above to switch to the enchantment UI, this part switches back (because it stays open after casting an echant, and would block other spell casts)